genny-project/alyson-v3
# Feature gen 1357

https://outcomelife.atlassian.net/browse/GEN-1357

## Bug Description:
Updating sidebar to be able to handle props for specific sizes.

## How to Test:
1) In `sidebar.default.json`, add the following code to the top level object:

`
 "sizeOnlyProps": {
    "sm": {
      "closeOnItemClick": true
    }
  }
`

2) Resize the window to be small (less than 767 px wide).
3) Open the sidebar and click on an item. The sidebar should close.
4) Resize the window to fill the screen.
5) Repeat step 3. Sidebar should remain open.
